frontend/packages/shared/src/components/FormField/FormField.tsx

```diff
@@ -151,6 +153,9 @@ export const FormField = <T extends unknown, TT extends unknown>({
               onChange: handleOnChange,
               customRequired: isRequired,
             }),
+            ({ component: ChildrenComponent, ...props }) => (
+              <ChildrenComponent {...props} />
+            ),
```

Comment:
Selv om `cloneElement` ikke brukes direkte her, er det teknisk sett det samme som skjer i denne koden. Denne saken innebærer mer refaktorering. Det enkleste er nok å lage en lage en render prop i stedet for å bruke `children`, som forklart her: https://react.dev/reference/react/cloneElement#passing-data-with-a-render-prop

På den måten oppnår vi at at det ikke er noen skjult manipulasjon av komponentene. Hvis vi tar `EditNumberValue` som eksempel, vil det se omtrent slik ut:
```
    <FormField
      id={component.id}
      value={component[propertyKey]}
      onChange={handleValueChange}
      propertyPath={component.propertyPath}
      helpText={helpText}
      renderField={(props) => (
        <StudioDecimalInput
          {...props}
          description={getComponentPropertyLabel(propertyKey, t)}
        />
      )}
    />
```
Her er det helt tydelig at `StudioDecimalInput` får props fra `FormField`, og vi kan være trygge på at komponenten blir lastet akkurat slik vi har spesifisert.

Reply:
FormField  ble brukt i 38 steder, jeg endret alt , men tenkte å vente med å fikse tester til at du se på endringene jeg har gjort.